pkg/views/workspace/info/view.go

```diff
@@ -99,6 +99,8 @@ func getSingleProjectOutput(project *apiclient.Project, isCreationView bool) str
 		output += getInfoLineGitStatus("Branch", &project.State.GitStatus) + "\n"
 	}
 
+	output += getInfoLinePrNumber(project.Repository.PrNumber, project.Repository.Branch, project.State.GitStatus.CurrentBranch)
```

Comment:
We need a nil check here (and below).

If the State is nil, then we display the PR as well.

Reply:
done